app/_locales/en/messages.json

```diff
@@ -1116,8 +1116,12 @@
   "deleteNetwork": {
     "message": "Delete network?"
   },
-  "deleteNetworkDescription": {
-    "message": "Are you sure you want to delete this network?"
+  "deleteNetworkIntro": {
+    "message": "If you delete this network, you will need to add it again to view your assets in this network."
```

Comment:
According to our content guidelines, single sentences should not have a period. cc @coreyjanssen 
```suggestion
    "message": "If you delete this network, you will need to add it again to view your assets in this network"
```
https://www.notion.so/MetaMask-Content-Design-Style-Guide-7d0b7579786540afbe02780ebb203254?pvs=4#7ffcee3343bc4115991c7a7e0d2f1c71

![Screenshot 2023-08-02 at 2 06 32 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/d0dbfcfb-5c01-49fa-9e5d-115d1bfa974b)